stdlib/public/core/UnsafeBufferPointer.swift.gyb

```diff
@@ -366,6 +389,141 @@ public struct Unsafe${Mutable}BufferPointer<Element>
   public func makeIterator() -> UnsafeBufferPointerIterator<Element> {
     return UnsafeBufferPointerIterator(_position: _position, _end: _end)
   }
+  
+  /// Deallocates the heap block previously allocated at this buffer pointer’s 
+  /// base address. 
+  ///
+  /// This buffer pointer’s `baseAddress` must be `nil` or a pointer to a previously 
```

Comment:
These are nicely documented!

Small nit here and elsewhere below: could you please straighten the apostrophe? I believe it interferes with some downstream tooling and have seen others go through and do this sort of clean-up; might as well have it straightened in the first place.